<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWidget</class>
 <widget class="QDialog" name="SignupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,1,2,1,2,1">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="loginLabel">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="loginLineEdit">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="passwordLabel">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="signupPushButton">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
